asserts/ifacedecls.go

```diff
@@ -220,7 +222,256 @@ func compileAttributeConstraints(constraints interface{}) (*AttributeConstraints
 	return &AttributeConstraints{matcher: matcher}, nil
 }
 
+type fixedAttrMatcher struct {
+	result error
+}
+
+func (matcher fixedAttrMatcher) match(context string, v interface{}) error {
+	return matcher.result
+}
+
+var (
+	AlwaysMatchAttributes = &AttributeConstraints{matcher: fixedAttrMatcher{nil}}
+	NeverMatchAttributes  = &AttributeConstraints{matcher: fixedAttrMatcher{errors.New("not allowed, never matching")}}
```

Comment:
I'm not sure myself atm, may be safer indeed to just make it "now allowed" for now